src/cli/bin.js

```diff
@@ -2,11 +2,19 @@
 
 'use strict'
 
+const pipe = require('pipe-args')
 const yargs = require('yargs')
 const updateNotifier = require('update-notifier')
 const readPkgUp = require('read-pkg-up')
 const utils = require('./utils')
 
+const enableStdin = [
+  'data', 'path', 'object data', 'ref', 'key', 'ipfs-path', 'add', 'get', 'cat',
+  'name', 'address', 'files', 'peer', 'recursive', 'default-config', 'peer ID'
```

Comment:
Seems to be missing a few commands. `config` and `pubsub` are two I remember from the top of my head, but I guess there is more.

Reply:
Here is a Github search that shows which `go-ipfs` commands accepts stdin: https://github.com/ipfs/go-ipfs/search?utf8=%E2%9C%93&q=EnableStdin&type=